docs/HowToArchiveJournal.md

```diff
@@ -17,13 +17,134 @@ For more info on NuoDB database journaling, please consult the official [NuoDB d
 
 Since the `journal` has to write all commits durably to disk, the speed of the disk directly influences the commit latency.
 To achieve the best performance, NuoDB recommends placing the `journal` on the fastest disk available.
+In contract, the `archive` should be using the disk with the largest throughput.
 
 By default, the `journal` is located in a subdirectory of the `archive`.
 To achieve the best cost vs speed tradeoff, you can separate the journal from the archive.
 To achieve that set the database helm template values `database.sm.noHotCopy.journalPath.enabled` and `database.sm.hotCopy.journalPath.enabled` to `true` and configure it with the desired persistence settings.
 
 Kubernetes stateful sets volume mounts are immutable and as such, the setting can not be changed easily on an existing database.
-More on upgrades below [Upgrading existing domains](#upgrading-existing-domains).
 
 ## Upgrading existing domains
-TODO
\ No newline at end of file
+
+### Overview
+
+StatefulSet volume mounts are immutable and any attempts to change this value will result in the following error:
+```
+Error: UPGRADE FAILED: cannot patch "sm-database-nuodb-cluster0-demo-hotcopy" with kind StatefulSet: \
+StatefulSet.apps "sm-database-nuodb-cluster0-demo-hotcopy" is invalid: spec: Forbidden: \
+updates to statefulset spec for fields other than 'replicas', 'template', and 'updateStrategy' are forbidden
+```
+
+To change the domain from using `an in-archive journal` to an `a separated journal` or vice versa, Kubernetes requires the StatefulSet to be deleted and recreated with the new settings.
+
+NuoDB Helm Charts offer two distinct Storage Manager StatefulSets.
+One StatefulSet controls Storage Managers which have a `backup` directory and are referred to as `HotCopy SMs`.
+The second StatefulSet which does not have such a directory, and we call Storage Managers in this SS `NoHotCopy SMs`.
```

Comment:
One use of StatefulSet is referenced as "SS".  It's not obvious to me why that is. 
Suggest either use the familiar (SS) after the first use of StatefulSet, or convert this occurrence to a spelled out notation.